modules/heat_transfer/doc/content/source/functormaterials/RadiativeP1DiffusionCoefficientMaterial.md

```diff
@@ -0,0 +1,23 @@
+# (AD)RadiativeP1DiffusionCoefficientMaterial
+
+This [functor material](/FunctorMaterials/index.md) computes the effective
+P1 radiative heat transfer coefficient as follows:
+
+\begin{equation}
+\Gamma = \frac{1}{3 \kappa + \sigma_{s, eff}}
+\end{equation}
+
+where
+
+- $\kappa$ is the opacity (SI units $m$)
+- $\sigma_{s, eff}$ is the P1 effective scattering cross section (SI units $m$), e.g., $3 \sigma_{tr}$ with $\sigma_{tr}$ being the transport cross section.
+
+!alert warning
+If scattering in the medium can be assumed isotropic,
+then $\sigma_{s, eff} \approx 0$ is a good approximation.
```

Comment:
This is correct, $\sigma_{s, eff} = 3 \sigma_{tr}$. However, $\sigma_{tr}$ is the P1 transport cross section, which is defined as $\sigma_{tr} = \sigma_s (1 - <\mu>)$, where $<\mu>$ is the average scattering cosine angle of the scattering collision. Note that if the scattering is isotropic, then all output directions are equally probable and, hence, $<\mu> = \int_{-1}^1 P(\mu) \mu d \mu = \int_{-1}^1 1 \mu d \mu = 0$

Reply:
Added incropera as a reference.

Reply:
Oh OK, so when the medium is isotropic, then $\sigma_{s,eff}=3\sigma_{tr}=3\sigma_s$ (since $<\mu>=0$). 
I see that the warning is now gone, so I'm good with the current version of the documentation. 


Thanks for spelling it out for me!

Reply:
Yes, you are write. I will delete this warning

Reply:
Better now?

Reply:
Yes, thank you!